spec/services/candidate_interface/existing_candidate_authentication_spec.rb

```diff
@@ -4,8 +4,25 @@
   include CourseOptionHelpers
 
   describe '#execute' do
+    context 'when the candidate already has 3 application choices' do
+      it 'sets the candidates course_from_find_id to nil and sets candidate_already_has_3_courses too true' do
+        new_course = create(:course)
+        candidate = create(:candidate, course_from_find_id: new_course.id)
+        create(:completed_application_form, candidate: candidate, application_choices_count: 3)
+
+        service = described_class.new(candidate: candidate)
+        service.execute
+
+        expect(service).to be_candidate_already_has_3_courses
+        expect(service).not_to be_candidate_has_new_course_added
```

Comment:
Not a big fan of this syntax, but Rubcop was complaining 🤷‍♂

Reply:
I personally find `expect(service.candidate_already_has_3_courses?).to be_truthy`

Reply:
Yeah, agreed, "to be candidate has new course added" totally salad like sounds weird a word.

Can you turn off the cop?

Reply:
Have done!